submissions/askuznetsov/html-css-popup/style.scss

```diff
@@ -0,0 +1,229 @@
+$color-main: #d0d0ce;
+$color-secondary: royalblue;
+$border-main: 2px solid $color-secondary;
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+body {
+  background-color: $color-main;
+  box-sizing: border-box;
+}
+
+.header {
+  margin: 24px 24px;
+
+  &__nav {
+    display: block;
+    padding: 8px;
+
+    &__list {
+      list-style-type: none;
+      display: flex;
+      justify-content: space-around;
+      align-items: center;
+      flex-wrap: wrap;
+
+      &__item {
+        height: 24px;
+        margin: 24px;
+
+        &:hover {
+          cursor: pointer;
+        }
+
+        &__image {
+          vertical-align: middle;
+          padding-bottom: 4px;
+          height: inherit;
+        }
+      }
+    }
+  }
+}
+
+@media only screen and (max-width: 1200px) {
+  .header {
+    &__nav {
+      &__list {
+        justify-content: flex-start;
+      }
+    }
+  }
+}
+
+.title {
+  font-size: 24px;
+  line-height: 14px;
+}
+
+.arrow {
+  width: 12px;
+  height: 12px;
+  background-color: white;
+  position: absolute;
+  margin-left: -12px;
+  margin-top: -30px;
+  transform: rotate(45deg);
+}
+
+.popup {
+  margin-top: 8px;
+  background-color: white;
+  padding: 24px;
+  border-radius: 3px;
+
+  display: none;
+}
+
+.menu {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+
+  &__item {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    width: 33%;
+    margin-bottom: 24px;
+
+    &__link {
```

Comment:
Please, add the default width of the border for this element because I see a layout shift when the user is trying to hover over this element. These styles should fix this issue:
```css
border: 2px solid transparent;
```

Reply:
Fixed.